packages/calcite-components/src/components/tooltip/tooltip.tsx

```diff
@@ -158,12 +157,14 @@ export class Tooltip implements FloatingUIComponent, OpenCloseComponent {
   //
   // --------------------------------------------------------------------------
 
-  connectedCallback(): void {
-    this.setUpReferenceElement(this.hasLoaded);
+  async connectedCallback(): Promise<void> {
+    await componentOnReady(this.el);
+    this.setUpReferenceElement(true);
+    connectFloatingUI(this, this.effectiveReferenceElement, this.el);
```

Comment:
this.setUpReferenceElement calls connectFloatingUI so this can be removed.

Reply:
Oh, I missed this one. Fixing, thanks!